.golangci.yml

```diff
@@ -0,0 +1,28 @@
+run:
+  deadline: 5m
+
+linters:
+  disable-all: true
+  enable:
+    - errcheck
+    # - errorlint
+    # - goconst
+    # - gocyclo
+    # - gofmt
+    # - goimports
+    # - gosec
+    # - gosimple
+    # - govet
+    # - ineffassign
+    # - misspell
+    # - nakedret
+    # - nilerr
+    # - prealloc
+    # - revive
+    # - staticcheck
+    # - typecheck
+    # - unconvert
+    # - unused
+    # - whitespace
```

Comment:
The number of errors is high and all of it in a single PR can become cumbersome to review. I'll open follow-up PRs to uncomment these along with fixes in multiple PRs making it easier for review.

We can use https://github.com/deislabs/ratify/issues/546 to track the status